### DISCUSSION TITLE
Add support for exporting animation as a video .

### DISCUSSION THREAD

**Discussion by ghost** [godotengine/godot-proposals#5851] [General Discussions]
Please add a feature for saving animation as a video on android . this system gonna help developer to create more games and generate more ideas  of creating apps and tools . and if the video saving format is .mp4 then this gonna help the user to share the video. please developer implement this feature

**Comment by Calinou** [comment]
This is already available in 4.0.beta: https://godotengine.org/article/movie-maker-mode-arrives-in-godot-4

There are no plans to backport this to 3.x though.
